packages/fast-components-styles-msft/src/call-to-action/index.ts

```diff
@@ -105,10 +113,21 @@ const styles: ComponentStyles<CallToActionClassNameContract, DesignSystem> = {
         marginRight: directionSwitch("", "6px"),
         transform: directionSwitch("none", "rotate(180deg)"),
         transition: "all 600ms cubic-bezier(0.19, 1, 0.22, 1)",
+        ...highContrastForeground,
     },
     callToAction__primary: {
         "& $callToAction_glyph": {
             fill: accentForegroundCut,
+            [highContrastSelector]: {
+                fill: "HighlightText !important",
+            },
+        },
+        "&:hover": {
+            "& $callToAction_glyph": {
```

Comment:
instead of doing a nested object you can combine these to `"&:hover $callToAction_glyph": {`

Reply:
combined